scripts/beekeeper.pl

```diff
@@ -284,6 +286,15 @@ sub main {
 
     if($reset_job_id) { $queen->reset_job_by_dbID_and_sync($reset_job_id); }
 
+    if($reset_job_for_input_id and $self->{'analyses_pattern'}) {
+        my @analyses_list = $self->{'pipeline'}->collection_of('Analysis')->find_all_by_pattern( $self->{'analyses_pattern'});
```

Comment:
`find_all_by_pattern` returns an array-_ref_. Actually, I'd also prefer passing a reference to `reset_job_by_input_id_and_sync` rather than the expanded list. It makes it easier to add extra arguments if we need to